python/semantic_kernel/connectors/ai/hugging_face/services/hf_text_completion.py

```diff
@@ -9,7 +9,7 @@
 
 from semantic_kernel.connectors.ai.ai_exception import AIException
 from semantic_kernel.connectors.ai.ai_service_client_base import AIServiceClientBase
-from semantic_kernel.connectors.ai.hugging_face.hf_request_settings import (
```

Comment:
All the other imports are using the absolute name with `semantic_kernel.`, shouldn't this follow the same?